features/api_configure_retry_service.feature

```diff
@@ -18,6 +18,8 @@ Feature: api.u.pro.attach.auto.configure_retry_service
         """
         When I run `systemctl enable apitest.service` with sudo
         When I reboot the machine
+        # Cloud init may take a while here
+        And I wait `15` seconds
```

Comment:
Is there a reason for a 15 second wait or it's just the better option given the amount of time cloud-init takes?

Reply:
completely arbitrary
I am sad about this change because of the complex thought process on deciding the time: "on my machine it takes about 10 seconds, so I will set is to 15". 
And why does it take sooo long only on Bionic? Other series were fine...

Reply:
Makes sense, thanks!